demisto_sdk/commands/download/README.md

```diff
@@ -104,6 +110,7 @@ We assume that test playbooks contain the `test` word in their name.
 
 ### Not Supported
 Integrations / Scripts written in JavaScript.
+A playbook that starts with the word 'Test', it would be downloaded as a test playbook.
```

Comment:
we have 
```python
                    or name.startswith(("Test", "Test_", "test_", "Test-", "test-")):
```
is this documentation correct?

Reply:
yeah, any playbook starting with the word test in any form is considered as tpb